src/pageEditor/modals/addBrickModal/addBrickModalHelpers.ts

```diff
@@ -37,13 +38,17 @@ export function getItemKey({
   columnIndex,
   data: { brickOptions },
   rowIndex,
-}: ItemKeyInput): RegistryId | number | undefined {
+}: ItemKeyInput): RegistryId | number {
   const resultIndex = getFlatArrayIndex({ rowIndex, columnIndex });
   // Number of bricks for the last Grid row could be less than the number of columns
   // Returning the index here, ItemRenderer will render an empty cell
   if (resultIndex >= brickOptions.length) {
     return resultIndex;
   }
 
-  return brickOptions.at(resultIndex)?.value;
+  const key = brickOptions.at(resultIndex)?.value;
+
+  assertNotNullish(key, `Item key at resultIndex: ${resultIndex} is nullish`);
+
+  return key;
```

Comment:
Nit:

The value/key itself would not be nullish, it's the fact that the Array.at() call can return nullish. `value` field on `BrickSelectOption` is required.
```suggestion
  const item = brickOptions.at(resultIndex);

  assertNotNullish(item, `Item at resultIndex: ${resultIndex} is nullish`);

  return item.value;
```